Changelog updates (#1289)
zeroc-ice/ice: CHANGELOG-3.7.md

- Fixed a bug in `IceSSL` that would result in a partial chain when the intermediate CA certificates were not installed
  in the client certificate store and were provided by the peer. This only affects the Windows Schannel implementation.

- Fixed a bug in `IceUtil::Time::toString`that results in bogus result when the microseconds representation of `IceUtil::Time`
  didn't fit in a 32 bit word. Thanks Fabio Osorio for reporting this issue [#1283](https://github.com/zeroc-ice/ice/issues/1283)

- The C++ 11 mapping builds now uses the compiler default C++ mode  when the compilerdefault mode is greater or equal to the
  required C++11 mode.

- Fixed a bug in IceStorm tracing, the subscribers collection was not lock while it was being traced and can be modified
  concurrently. Thanks to Bill Stoffel for reporting the issue and creating a pull requests [#1259](https://github.com/zeroc-ice/ice/pull/1259)

## Java Changes

- Fixed a race condition in IceBT transport for Android that can result in a `NullPointerException` thanks Marco Pfattner for
  reporting this issue [#1262](https://github.com/zeroc-ice/ice/issues/1262).

## JavaScript Changes

> - Fixed a bug in JavaScript mapping reading of interface by value arguments that can result in `Ice.UnexpectedObjectException` during
>   unmarshaling.

Reviewer: ```suggestion
- Fixed a bug in the JavaScript mapping where reading interface by value arguments could result in `Ice.UnexpectedObjectException` during
  unmarshaling.
```
